dask_cudf/io/csv.py

```diff
@@ -1,22 +1,36 @@
+import os
 from glob import glob
 
 import cudf
 from dask.base import tokenize
 from dask.compatibility import apply
 import dask.dataframe as dd
 
-
-def read_csv(path, **kwargs):
+def read_csv(path, chunksize=25e6, **kwargs):
```

Comment:
I recommend two things:

1.  Use `dask.utils.parse_bytes` on chunksize so users can providev values like `"25 MiB"`
2.  Use a larger blocksize.  I'll suggest "128 MiB", but @kkraus14 may have other suggestions

Reply:
Typically the larger the better, but "128 MiB" sounds like a good starting point.